tests/test_0016_nop_filter.py

```diff
@@ -1,7 +1,10 @@
+import unittest
 from pbtests.packetbeat import TestCase
 
 
 class Test(TestCase):
+    @unittest.skip(
+        "Disabled until filters have been added to processing chain again")
```

Comment:
Can you open an issue for this in case it is not done directly? Otherwise we probably forget about it.